src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'lock':
       return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
 
+    case 'garage_door':
+      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
```

Comment:
Well…

Maybe you can differentiate per domain like you do for the entity toggle?